style.css

```diff
@@ -0,0 +1,135 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+body {
+    font-family: Arial, sans-serif;
+    background-color: #f0f4f8;
+    color: #333;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    min-height: 100vh;
+}
+
+header {
+    background-color: #4a90e2;
+    color: white;
+    width: 100%;
+    padding: 1em 0;
+    text-align: center;
+}
+
+h1 {
+    margin: 0;
+}
+
+main {
+    display: flex;
+    justify-content: center;
+    align-items: flex-start;
+    background-color: #f0f4f8;
+    padding: 2em;
+    gap: 2em;
+    flex-wrap: wrap;
```

Comment:
Why are you applying `flex-wrap`?

Reply:
because it allows the elements inside the flex to wrap together.